storybook/src/components/FieldSet/FieldSet.docs.mdx

```diff
@@ -24,7 +24,15 @@ and provide the `id` of the describing element as its value.
 
 <Canvas of={FieldSetStories.WithDescription} />
 
-## With Error
+### With Not Required Label
```

Comment:
We can use ‘Optional’ instead of ‘Not Required’ in our docs and code. Just not for users.

And in this case, ‘Optional’ is enough instead of ‘With Optional Label’.